Fix octoberSky test input.txt

kovalromank opened the pull request.

kovalromank commented:
I think I fixed the input.txt file but idk how to verify it. Also the tests need to be updated because they fail with the fixed input.txt

ajayyy commented:
To fix the tests, you can run them locally (`cd tests/octoberSky; bash run_test.sh`), delete `output-expected` and then rename `output` to `output-expected`. I guess you can make sure your changed to `output-expected` only the row you changed in `input.txt`

kovalromank commented:
I believe the issue was that for some columns it wouldn't write anything and only a comma. Fixed it by writing a 0 followed by a comma and updated the expected logs
